app/test_main.py

```diff
@@ -1,3 +1,65 @@
 from app.main import get_human_age
+import pytest
+from typing import Any
 
-# write your code here
+
+class TestValues:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_value",
+        [
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="Correct values for 15, 15"),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="Correct values for, 24, 24"),
+            pytest.param(
+                28,
+                29,
+                [3, 3],
+                id="Correct values for 28, 29"),
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="Return 0 when animal age is 0"),
+            pytest.param(
+                100.5,
+                100.5,
+                [21, 17],
```

Comment:
The test case with inputs `(100.5, 100.5)` expects `[21, 17]`, but the task specifies that age should be a whole number. Consider adjusting your function to handle fractional inputs appropriately.